src/sentry/static/sentry/app/views/settings/components/forms/jsonForm.jsx

```diff
@@ -64,6 +65,7 @@ class JsonForm extends React.Component {
               <PanelBody>
                 {fields.map(field => (
                   <FieldFromConfig
+                    access={access}
```

Comment:
What does this do? It doesn't appear to be used in `FieldFromConfig`

Reply:
access is a Set of permissions for user, it's used in a callback (currently only for `disabled`).